src/slimevr101.md

```diff
@@ -2,29 +2,72 @@
 
 ## What is SlimeVR?
 
-SlimeVR is a low-cost solution to full-Body tracking in virtual reality. It uses forward kinematics to build a model of your skeleton calculated from the rotation of each individual tracker, with your headset and controllers being the only absolute known position.
+SlimeVR is an affordable solution for full-body tracking in virtual reality. It uses forward kinematics[^note] to create a model of your body by calculating the position of each tracker based on its rotation. The only fixed point is your headset and controllers, which are used as reference positions.
```

Comment:
Controllers are by default not used as fixed positions, might be better to leave this out